stardis/io/model/marcs.py

```diff
@@ -30,6 +31,21 @@ def to_geometry(self):
         r = self.data.depth.values * u.cm
         return Radial1DGeometry(r)
 
+    # def to_composition(self):
```

Comment:
add something like "this will be used later" ?

Reply:
This is a reasonable suggestion that I don't disagree with on principle, but as soon as this gets merged I'll be uncommenting and hooking this method up with a branch that prepares the StellarModel class from a marcs model. I'd rather not go back and deal with the merge conflicts on that branch to add this comment that will be short-lived.